### PR TITLE
Add metrics export subcommand to output CSV file of host metrics

### PR DESCRIPTION
**Pull request by denniskuczynski**
This adds a metric subcommand with nested export command that writes all host metrics (with default settings) to standard out as CSV data.

(Needs lots of cleanup)

Example

```
denniskuczynski@dennismac mops-cli (5_metrics)$ ./bin/mops metrics export -g 55807ce0e4b0ce4c2b652f48 -h 3927a5536e36ce4885424e382f9f0b2f
Date,ASSERT_MSG,ASSERT_REGULAR,ASSERT_USER,ASSERT_WARNING,BACKGROUND_FLUSH_AVG,CACHE_BYTES_READ_INTO,CACHE_BYTES_WRITTEN_FROM,CACHE_DIRTY_BYTES,CACHE_USED_BYTES,COMPUTED_MEMORY,CONNECTIONS,CURSORS_TOTAL_CLIENT_CURSORS_SIZE,CURSORS_TOTAL_OPEN,CURSORS_TOTAL_TIMED_OUT,EFFECTIVE_LOCK_PERCENTAGE,EXTRA_INFO_PAGE_FAULTS,GLOBAL_ACCESSES_NOT_IN_MEMORY,GLOBAL_LOCK_CURRENT_QUEUE_READERS,GLOBAL_LOCK_CURRENT_QUEUE_TOTAL,GLOBAL_LOCK_CURRENT_QUEUE_WRITERS,GLOBAL_PAGE_FAULT_EXCEPTIONS_THROWN,INDEX_COUNTERS_BTREE_ACCESSES,INDEX_COUNTERS_BTREE_HITS,INDEX_COUNTERS_BTREE_MISSES,INDEX_COUNTERS_BTREE_MISS_RATIO,JOURNALING_COMMITS_IN_WRITE_LOCK,JOURNALING_MB,MEMORY_MAPPED,MEMORY_RESIDENT,MEMORY_VIRTUAL,MUNIN_CPU_IOWAIT,MUNIN_CPU_IRQ,MUNIN_CPU_NICE,MUNIN_CPU_SOFTIRQ,MUNIN_CPU_STEAL,MUNIN_CPU_SYSTEM,MUNIN_CPU_USER,NETWORK_BYTES_IN,NETWORK_BYTES_OUT,NETWORK_NUM_REQUESTS,OPCOUNTERS_CMD,OPCOUNTERS_DELETE,OPCOUNTERS_GETMORE,OPCOUNTERS_INSERT,OPCOUNTERS_QUERY,OPCOUNTERS_UPDATE,OPLOG_MASTER_TIME,OPLOG_RATE_GB_PER_HOUR,TICKETS_AVAILABLE_READS,TICKETS_AVAILABLE_WRITES
2015-07-20T19:28:00.000Z,,,,,,,,,,570,10,0,0,,,,,0,0,0,,,,,,,,4174,58,8918,,,,,,,,,,,,,,,,,0,,,
2015-07-20T19:29:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,639.1090909090909,4486.672727272728,8.127272727272727,7.327272727272727,0,0,0,0.6181818181818182,0.18181818181818182,0,,,
2015-07-20T19:30:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,,,4174,58,8918,,,,,,,,712.6363636363636,5238.327272727272,9.309090909090909,8.363636363636363,0,0,0,0.7272727272727273,0.21818181818181817,0,,,
2015-07-20T19:31:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,640.0727272727273,4487.072727272727,8.145454545454545,7.345454545454546,0,0,0,0.6181818181818182,0.18181818181818182,0,,,
2015-07-20T19:32:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,712.6363636363636,5238.327272727272,9.309090909090909,8.363636363636363,0,0,0,0.7272727272727273,0.21818181818181817,0,,,
2015-07-20T19:33:00.000Z,0,0,0,0,0.5,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,672.6181818181818,4843.718181818182,8.663636363636364,7.790909090909091,0,0,0,0.6727272727272727,0.2,0,,,
2015-07-20T19:34:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,,,4174,58,8918,,,,,,,,649.6545454545454,4562.581818181819,8.309090909090909,7.509090909090909,0,0,0,0.6181818181818182,0.18181818181818182,0,,,
2015-07-20T19:35:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,699.8909090909091,5163.836363636364,9.090909090909092,8.145454545454545,0,0,0,0.7272727272727273,0.21818181818181817,0,,,
2015-07-20T19:36:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,631.3275862068965,4362.5,8.017241379310345,7.258620689655173,0,0,0,0.5862068965517241,0.1724137931034483,0,,,
2015-07-20T19:37:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,675.6730769230769,4829.076923076923,8.73076923076923,7.884615384615385,0,0,0,0.6538461538461539,0.19230769230769232,0,,,
2015-07-20T19:38:00.000Z,0,0,0,0,0,,,,,570,9,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,703.1272727272727,5179.963636363636,9.145454545454545,8.2,0,0,0,0.7272727272727273,0.21818181818181817,0,,,
2015-07-20T19:39:00.000Z,0,0,0,0,0,,,,,570,10,0,0,0,,0,,0,0,0,,,,,,0,0,4174,58,8918,,,,,,,,640.0727272727273,4503.818181818182,8.145454545454545,7.345454545454546,0,0,0,0.6181818181818182,0.18181818181818182,0,,,
```

### CODE REVIEW

File: bin/mops [hunk 1]
```diff
@@ -7,5 +7,6 @@ var program = require('commander');
 program
   .version('0.0.0')
   .command('login [name]', 'Store MMS username and api key.')
+  .command('metrics', 'Metrics subcommand')
```
Comment (PeaSeaGee) [inline bin/mops]: This version does not seem to include the corrected helper for topology, nor the alerts exporting.

Comment (denniskuczynski) [inline bin/mops]: yeah, i'll merge with the latest now

**Comment by denniskuczynski** [comment]
#5

**Comment by ohEmily** [comment]
LGTM :+1:

**Comment by denniskuczynski** [comment]
master @ d0947e4b69c6a4f536d0afdd3ec64a15dd6d4f0e

**Pull request closed without merging**